packages/contourpy/patches/0001-Install-pybind11-from-github-repo-after-PR-4767.patch

```diff
@@ -0,0 +1,25 @@
+From d5e101ce4575125727cfa5fc2a9cbb829637beab Mon Sep 17 00:00:00 2001
+From: Ian Thomas <ianthomas23@gmail.com>
+Date: Sun, 12 Nov 2023 14:34:25 +0000
+Subject: [PATCH] Install pybind11 from github repo after PR 4767
+
```

Comment:
> To use pybind11 with clang >= 17 we need https://github.com/pybind/pybind11/pull/4767. This isn't in a pybind11 release yet, so I have included a patch to install pybind11 direct from the github commit that includes the PR. This patch can be removed after the next pybind11 release.

Can you add a comment in the patch message here?

Reply:
Done.